src/instr/instr_private.h

```diff
@@ -72,10 +72,10 @@ class s_type :public ess_type {
   e_entity_types kind;
   xbt_dict_t children;
   xbt_dict_t values; //valid for all types except variable and container
+  s_type(const char *typeNameBuff, const char *key, const char *color, e_entity_types kind, type_t father);
```

Comment:
I'm happy you're working to cleanup the s_type but it should not be part of the same PR.